extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -0,0 +1,37 @@
+# 3DTILES_implicit_tiling
+
+**Version 0.0.0**, December 1, 2020
+
+## Contributors
+
+* Sam Suhag, Cesium
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Erixen Cruz, Cesium
```

Comment:
Just a note, we should double check the order of authors in a future discussion once the spec is mostly written

Reply:
is there a convention for ordering? i was thinking alphabetical by last name maybe

Reply:
We list authors by how much a person contributes to the schema/spec.